tests/cli/clean/conftest.py

```diff
@@ -1,14 +1,88 @@
 """Fixtures for cli clean tests"""
-
+from pathlib import Path
 import pytest
+import datetime as dt
+
+from cg.meta.workflow.balsamic import BalsamicAnalysisAPI
+from cg.apps.balsamic.api import BalsamicAPI
+
+
+@pytest.fixture
+def balsamic_clean_store(base_store, helpers):
+    store = base_store
+    timestamp_yesterday = dt.datetime.now() - dt.timedelta(days=1)
```

Comment:
Hi, could you perhaps add this [here](https://github.com/Clinical-Genomics/cg/blob/4e1e1ea8cefee760b7d34ad2912525933b71b9fc/tests/conftest.py#L307) and make it a general fixture and add a today? Something like

```python
@pytest.fixture(scope="function", name="today")
def fixture_today() -> dt.datetime:
    """Return a time stamp of todays date in date time format"""
    return dt.datetime.now()

@pytest.fixture(scope="function", name="yesterday")
def fixture_yesterday(today) -> dt.datetime:
    """Return a time stamp of yesterdays date in date time format"""
    return today - dt.timedelta(days=-1)
```

Reply:
I guess that could be useful in many cases

Reply:
Will do!